src/api/app/controllers/webui/project_controller.rb

```diff
@@ -448,6 +448,8 @@ def monitor
         @repohash[repo].delete(arch) unless has_packages
       end
     end
+
+    @repoarray = @repohash.sort
```

Comment:
You know what makes me happy? Telling you now that if repohash is not used in the views anymore, we should better not make it instance variable to make this clear :)

But I wonder: app/mixins/webui/load_buildresults.rb and spec/controllers/webui/project_controller_spec.rb access this variable. I guess this is obsolete

Reply:
and we use it in different methods. We would have to return it in the method and there are other variables there we are not returning. I don't think this is that important. :thinking:

Reply:
You know how this works when you restart optimizing code, no? :)

![getting into refactoring](https://media.giphy.com/media/Mdct4RNEkGIrm/giphy.gif)

Reply:
The problem is if you don't change the name, but don't change the users from expecting a hash either you get a problem just as well

Reply:
it is changed! :wink: